deepchem/utils/poly_wd_graph_utils.py

```diff
@@ -0,0 +1,132 @@
+from typing import List, Tuple
+import re
+
+
+class PolyWDGStringValidator():
+    """
+    Class for validating the string format of weighted directed graph data
```

Comment:
You need to document what this string format is with examples and references in the docs